### PR TITLE
K8SPSMDB-370 Change messages for meta waitining

### PR DESCRIPTION
**Pull request by nonemax**
[K8SPSMDB-370](https://jira.percona.com/browse/K8SPSMDB-370) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

There are some confusing messages in the operator log when backing up and restoring the cluster, but both operations are successful. The reason why there is this messages it because PBM need some time for syncing and start work. So we dont have any meta information about backup or restore before PBM starts

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| demand-backup | failed |
| self-healing | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| scheduled-backup | failed |
| version-service | passed |
| users | passed |
| demand-backup-sharded | failed |
| upgrade | passed |
| upgrade-sharded | passed |

**Review by dAdAbird — DISMISSED** [review]

File: pkg/controller/perconaservermongodbbackup/backup.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func (b *Backup) Status(cr *api.PerconaServerMongoDBBackup) (api.PerconaServerMo
 		return status, errors.Wrap(err, "get pbm backup meta")
 	}
 	if meta == nil || meta.Name == "" {
-		log.Info("No backup found", "PBM name", cr.Status.PBMname, "backup", cr.Name)
+		log.Info("Waiting for backup meta", "PBM name", cr.Status.PBMname, "backup", cr.Name)
```
Comment (spron-in) [inline pkg/controller/perconaservermongodbbackup/backup.go:93]:
```suggestion
		log.Info("Waiting for backup metadata", "PBM name", cr.Status.PBMname, "backup", cr.Name)
```

File: pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ func (r *ReconcilePerconaServerMongoDBRestore) reconcileRestore(cr *psmdbv1.Perc
 	}
 
 	if meta == nil || meta.Name == "" {
-		log.Info("No restore found", "PBM name", cr.Status.PBMname, "restore", cr.Name, "backup", cr.Spec.BackupName)
+		log.Info("Waiting for restore meta", "PBM name", cr.Status.PBMname, "restore", cr.Name, "backup", cr.Spec.BackupName)
```
Comment (spron-in) [inline pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go:195]:
```suggestion
		log.Info("Waiting for restore metadata", "PBM name", cr.Status.PBMname, "restore", cr.Name, "backup", cr.Spec.BackupName)
```

**Review by spron-in — CHANGES_REQUESTED** [review]

File: pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go (hunk 2 above)
Comment (nonemax) [reply pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go]: done https://github.com/percona/percona-server-mongodb-operator/pull/536/commits/3bf8a664776bed687b6d30dff9e6be5f6b227a69

File: pkg/controller/perconaservermongodbbackup/backup.go (hunk 1 above)
Comment (nonemax) [reply pkg/controller/perconaservermongodbbackup/backup.go]: done https://github.com/percona/percona-server-mongodb-operator/pull/536/commits/3bf8a664776bed687b6d30dff9e6be5f6b227a69

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-536-58a95e62

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| rs-shard-migration | passed |
| storage | passed |
| self-healing | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-536-3bf8a664

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| one-pod | passed |
| demand-backup | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| demand-backup-sharded | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |

**Review by delgod — APPROVED** [review]

**Review by spron-in — APPROVED** [review]
lgtm

**Review by dAdAbird — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| security-context | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| demand-backup-sharded | failed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |

**Pull request merged**